src/include/hook_func.h

```diff
@@ -71,6 +71,7 @@ extern "C" {
 struct mom_hook_action {
 	char            hookname[PBS_HOOK_NAME_SIZE];
 	unsigned int	action;
+	unsigned int	reply_expected; /* reply expected from mom for sent out actions */
```

Comment:
Can you tell me exactly the values that this take? I scanned through server/hook_func.c, and I only see this set (or unset) as:
pact->reply_expected = 0U;
pact->reply_expected &= ~(event);

Reply:
Thank you for flagging this. I must have inadvertently removed some lines while reverting some debug lines. Now I have carefully gone through my development branch and added back the missing lines in https://github.com/openpbs/openpbs/pull/1992/commits/d54ce17afb4e9d6cd1a3da070504726722e60e5c